patches/chrome-android-BUILD.gn.patch

```diff
@@ -43,7 +43,15 @@ index dd59ffa4ffb9bd12d28e10aa64dc7dcfdabc0849..dd6cff83b144e5e7a572b97d883e4db6
  }
  
  action("chrome_android_java_google_api_keys_srcjar") {
-@@ -1392,6 +1396,7 @@ if (current_toolchain == default_toolchain) {
+@@ -690,6 +694,7 @@ junit_binary("chrome_junit_tests") {
+   data_deps = [ "//testing/buildbot/filters:chrome_junit_tests_filters" ]
+ 
+   package_name = chrome_public_manifest_package
++  assert_no_deps -= [ "//content", "//mojo/public/mojom/base", ]
```

Comment:
this seems like a real problem, not something we should remove. What was the build error?

Reply:
This could just be temporary to get the build not fail. Android work hasn't been finished yet. cc: @samartnik.

The error is:

Reply:
It seems like there was similar problem https://github.com/brave/brave-browser/issues/5123
I'm not quite sure how to workaround it at the moment

Reply:
I am not sure why `brave_generated_resources_grit` depends on `//brave/components/services:brave_content_manifest_overlays`.

Reply:
`//content` here is excessive, the problem is specifically with `//mojo/public/mojom/base`

Reply:
Android builds without issues without dep mentioned by @mkarolin 
@bridiver do you remember why it was added? Do we need it for other platforms?

Reply:
this is an old dep that should have been removed when the manifest overlays converted to c++

Reply:
Fixed in c281afeb7735b94ad0faffdfd6b8e26a35e80cee